extensions/wikia/RWEPageHeader/styles/_dropdown.scss

```diff
@@ -0,0 +1,61 @@
+@import 'variables';
+@import 'skins/shared/mixins/flexbox';
+@import 'skins/shared/mixins/transform';
+
+.rwe-page-header-nav__dropdown {
+	display: none;
+	margin-top: 13px;
+
+	.rwe-page-header-nav__dropdown-first-level {
+		@include display-inline-flex();
+		background-color: $background-color;
+		box-shadow: 0 $content-spacing-small $content-spacing-medium-minus 0 rgba(26, 26, 26, .3);
```

Comment:
can we extract this color to _variables.scss? within the _search.scss the same value for box-shadow is used